langchain/src/util/googlevertexai-webauth.ts

```diff
@@ -63,6 +96,7 @@ export class WebGoogleAuth implements GoogleAbstractedClient {
     const res = await fetch(opts.url, fetchOptions);
 
     if (!res.ok) {
+      console.error(res);
```

Comment:
Remove log?

Reply:
Oops. Yeah. This was left in while I was debugging why my auth with web wasn't working.